go.mod

```diff
@@ -31,7 +31,8 @@ require (
 	github.com/gorilla/websocket v1.4.1
 	github.com/gosimple/slug v1.4.2
 	github.com/grafana/grafana-plugin-model v0.0.0-20190930120109-1fc953a61fb4
-	github.com/grafana/grafana-plugin-sdk-go v0.16.0
+	github.com/grafana/grafana-plugin-sdk-go v0.17.1-0.20200304132035-b6b43f60bc5b
```

Comment:
This should be updated to point at a tag.

Reply:
We probably need to document how to work with modules and using a non tagged version of SDK.

I usually add a replace directive in go.mod:
`replace github.com/grafana/grafana-plugin-sdk-go => github.com/grafana/grafana-plugin-sdk-go <remote branch name>`
Then run `GO111MODULE=on go mod tidy` followed by `GO111MODULE=on go mod vendor`.

go.mod replace `<remote branch name>` should then automatically change to reflect head commit in that branch. 

This way the module version shouldn't need to change in require section.